### PR TITLE
Take 2 on fixing CS

### PR DESCRIPTION
**Pull request by lsolesen**

### CODE REVIEW

File: modules/checkout/src/Plugin/Commerce/CheckoutFlow/CheckoutFlowWithPanesBase.php [hunk 1]
```diff
@@ -350,7 +350,9 @@ protected function buildPaneRow(CheckoutPaneInterface $pane, array &$form, FormS
       $pane_row['#attributes']['class'][] = 'pane-configuration-editing';
 
       $pane_row['configuration'] = [
-        '#parents' => array_merge($form['#parents'], ['panes', $pane_id, 'configuration']),
+        '#parents' => array_merge(
```
Comment (bojanz) [inline modules/checkout/src/Plugin/Commerce/CheckoutFlow/CheckoutFlowWithPanesBase.php]: Why this change? It's less readable.

**Comment by lsolesen** [comment]
```
codio@total-patrol:~/workspace/motionsplan.dk/web/modules/contrib/commerce$ vendor/bin/phpcs m
odules                                                                                        

FILE: ...ommerce/modules/checkout/src/Annotation/CommerceCheckoutPane.php                     
----------------------------------------------------------------------                        
FOUND 2 ERRORS AFFECTING 2 LINES                                                              
----------------------------------------------------------------------                        
 41 | ERROR | Class property $default_step should use lowerCamel                              
    |       | naming without underscores                                                      
 50 | ERROR | Class property $wrapper_element should use lowerCamel                           
    |       | naming without underscores                                                      
----------------------------------------------------------------------                        

Time: 14.29 secs; Memory: 37.75Mb 
```

**Comment by lsolesen** [comment]
```
codio@total-patrol:~/workspace/motionsplan.dk/web/modules/contrib/commerce$ vendor/bin/phpcs m
odules                                                                                        

FILE: ...b/commerce/modules/checkout/src/CheckoutFlowPluginCollection.php                     
----------------------------------------------------------------------                        
FOUND 1 ERROR AFFECTING 1 LINE                                                                
----------------------------------------------------------------------                        
 42 | ERROR | Description for the @return value is missing                                    
----------------------------------------------------------------------                        


FILE: ...dules/contrib/commerce/modules/order/src/Event/LineItemEvent.php                     
----------------------------------------------------------------------                        
FOUND 1 ERROR AFFECTING 1 LINE                                                                
----------------------------------------------------------------------                        
 35 | ERROR | Description for the @return value is missing                                    
----------------------------------------------------------------------                        


FILE: .../modules/contrib/commerce/modules/order/src/Event/OrderEvent.php                     
----------------------------------------------------------------------                        
FOUND 1 ERROR AFFECTING 1 LINE                                                                
----------------------------------------------------------------------                        
 35 | ERROR | Description for the @return value is missing                                    
----------------------------------------------------------------------                        


FILE: ...s/contrib/commerce/modules/price/src/Event/NumberFormatEvent.php                     
----------------------------------------------------------------------                        
FOUND 1 ERROR AFFECTING 1 LINE                                                                
----------------------------------------------------------------------                        
 35 | ERROR | Description for the @return value is missing                                    
----------------------------------------------------------------------                        


FILE: ...ommerce/modules/product/src/Entity/ProductAttributeInterface.php                     
----------------------------------------------------------------------                        
FOUND 1 ERROR AFFECTING 1 LINE                                                                
----------------------------------------------------------------------                        
 15 | ERROR | Description for the @return value is missing                                    
----------------------------------------------------------------------                        


FILE: ...ommerce/modules/product/src/Event/ProductAttributeValueEvent.php                     
----------------------------------------------------------------------                        
FOUND 1 ERROR AFFECTING 1 LINE                                                                
----------------------------------------------------------------------                        
 35 | ERROR | Description for the @return value is missing                                    
----------------------------------------------------------------------                        


FILE: ...ules/contrib/commerce/modules/product/src/Event/ProductEvent.php                     
----------------------------------------------------------------------                        
FOUND 1 ERROR AFFECTING 1 LINE                                                                
----------------------------------------------------------------------                        
 35 | ERROR | Description for the @return value is missing                                    
----------------------------------------------------------------------                        


FILE: ...rib/commerce/modules/product/src/Event/ProductVariationEvent.php                     
----------------------------------------------------------------------                        
FOUND 1 ERROR AFFECTING 1 LINE                                                                
----------------------------------------------------------------------                        
 35 | ERROR | Description for the @return value is missing                                    
----------------------------------------------------------------------            
```

File: modules/checkout/src/Plugin/Commerce/CheckoutFlow/CheckoutFlowWithPanesBase.php (hunk 1 above)
Comment (lsolesen) [inline modules/checkout/src/Plugin/Commerce/CheckoutFlow/CheckoutFlowWithPanesBase.php]: https://github.com/drupalcommerce/commerce/pull/376/files#diff-6b216d983cf0a62e9c648934c5687456L76

File: modules/order/src/Event/LineItemEvent.php [hunk 2]
```diff
@@ -32,7 +32,9 @@ public function __construct(LineItemInterface $line_item) {
   /**
    * The line item the event refers to.
    *
+   * @codingStandardsIgnoreStart
    * @return \Drupal\commerce_order\Entity\LineItemInterface
```
Comment (haringsrob) [inline modules/order/src/Event/LineItemEvent.php:35]: Personally, I would not apply an Ignore here?

I think it's ok to say:

``` php
/**
   * The store the event refers to.
   *
   * @return \Drupal\commerce_store\Entity\StoreInterface
   *   The StoreInterface object.
   */
  public function getStore() {
    return $this->store;
  }
```

File: modules/checkout/src/CheckoutFlowPluginCollection.php [hunk 3]
```diff
@@ -39,7 +39,9 @@ public function __construct(PluginManagerInterface $manager, $instance_id, array
   /**
    * {@inheritdoc}
    *
+   * @codingStandardsIgnoreStart
    * @return \Drupal\commerce_checkout\Plugin\Commerce\CheckoutFlow\CheckoutFlowInterface
```
Comment (haringsrob) [inline modules/checkout/src/CheckoutFlowPluginCollection.php:42]: If it is {@inheritdoc}, do we still have to tell what we are returning?

File: modules/checkout/src/Annotation/CommerceCheckoutPane.php [hunk 4]
```diff
@@ -1,4 +1,5 @@
 <?php
+// @codingStandardsIgnoreFile
```
Comment (haringsrob) [inline modules/checkout/src/Annotation/CommerceCheckoutPane.php]: I think we can quickly refactor this.

Comment (lsolesen) [inline modules/checkout/src/Annotation/CommerceCheckoutPane.php]: I agree with your changes. I will fix up as soon as possibble.

man. 9. maj 2016 kl. 20.46 skrev Harings Rob <notifications@github.com>:

Comment (lsolesen) [inline modules/checkout/src/Annotation/CommerceCheckoutPane.php]: @bojanz Also said that this should be refactored. However, until it is done, I think this should just be kept in.

**Comment by lsolesen** [comment]
@haringsrob I've pushed a commit removing the @codingStandardsIgnoreStart and -End for @return values. Could you please review it? Thanks!

**Comment by haringsrob** [comment]
To me it looks fine, still not sure if we need the @return values.

This line: https://github.com/drupalcommerce/commerce/pull/376/files#diff-b07fb5de05d7364d1069d470a10223d9R101 looks a bit weird to me now, but I'm not sure if it is an issue.

Also about this change I'm unsure: https://github.com/drupalcommerce/commerce/pull/376/files#diff-c4c0b2b751d222724e64ba9329d543a5R126

I'll leave the second opinion to @bojanz.

**Comment by lsolesen** [comment]
@haringsrob  Could you point to the line numbers?

**Comment by haringsrob** [comment]
It should be highlighted in orange when you click the link:
schermafbeelding 2016-05-10 om 14 39 44

File: src/CredentialsCheckFlood.php [hunk 5]
```diff
@@ -123,7 +123,7 @@ protected function getAccountIdentifier($ip, $name) {
       $account_by_name = $storage->loadByProperties(['name' => $name]);
       $this->accounts[$name] = reset($account_by_name);
     }
-    /** @var \Drupal\Core\Session\AccountInterface $account **/
+    // @var \Drupal\Core\Session\AccountInterface $account
```
Comment (lsolesen) [inline src/CredentialsCheckFlood.php]: @haringsrob I agree this looks a little strange, but the previous was failed by the validator.

File: tests/src/Unit/AvailabilityManagerTest.php [hunk 6]
```diff
@@ -90,7 +97,9 @@ public function testCheck() {
     $result = $this->availabilityManager->check($entity, 1);
     $this->assertFalse($result, 'The checked entity is not available when a checker that returns FALSE applies');
 
-    $expectedCheckers = [$first_checker, $second_checker, $third_checker, $fourth_checker];
```
Comment (lsolesen) [inline tests/src/Unit/AvailabilityManagerTest.php:93]: This also fails the validator because the line is longer than 80. @haringsrob and @bojanz What do you want to do about it?

**Comment by lsolesen** [comment]
@bojanz The failing Travis test must be a fluke. It works here https://travis-ci.org/lsolesen/commerce/builds/129111112

File: src/CredentialsCheckFlood.php (hunk 5 above)
Comment (haringsrob) [inline src/CredentialsCheckFlood.php]: Hi, this should work:

Instead of:
`/** @var \Drupal\Core\Session\AccountInterface $account **/`
it should be a single star at end:
`/** @var \Drupal\Core\Session\AccountInterface $account */`

The way of commenting has an actual purpose : https://www.drupal.org/node/2305593

File: tests/src/Unit/AvailabilityManagerTest.php (hunk 6 above)
Comment (haringsrob) [inline tests/src/Unit/AvailabilityManagerTest.php:93]: Correct way would be:

``` php
$array = [
  'bar',
  'foo',
];
```

**Comment by lsolesen** [comment]
I pushed another commit fixing what @haringsrob pointed out. If you like, I can squash all the commits to one single commit. Or you can do that yourself @bojanz.

**Comment by bojanz** [comment]
Let's give up on fixing Drupal.Array.Array.LongLineDeclaration, all of the changes made to accomodate it have resulted in less readable code.

**Comment by lsolesen** [comment]
@bojanz I squashed the changes and rebased on latest 8.x-2.x. I hope that this is to your liking :)

**Comment by bojanz** [comment]
Rebased, expanded, committed in 3d324103488a3c9c4242c6e671a34319247ced82.
